mobify/platform-scaffold
# [IM Extensibility] Connector Extension Set up

This PR adds a connector-extension to the scaffold and sets up the extensibility code in the integration manager. *Note:* Proper test/sample code will be added in another PR as described in the JIRA tickets

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-78 & https://mobify.atlassian.net/browse/WEBDATA-79


## Changes
- Added registerConnectorExtension function to the integration manager
- Added custom folder/commands file to the integration manager
- Set up basic structure for connector extension folder
- Added sample custom command and command override

## How to test-drive this PR
- Run `npm run dev`
- Preview the [SFCC Site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Navigate to a PDP
- Update one of the variation selections (size or colour)
- Look in the console, you should see the text "Custom Command Dispatched"
- Click the add to cart button
- Look in the console, you should see the text "Add to Cart Override"


## To Do Before Merging:
- [x] Remove test code
- [x] Set connector back to Merlins
